code/game/gamemodes/revolution/revolution.dm

```diff
@@ -449,3 +456,49 @@
 	dat += "<HR>"
 
 	return dat
+
+
+/proc/revconvert(mob/living/user)
+	var/list/Possible = list()
+	for(var/mob/living/carbon/human/P in oview(user))
+		if((P.stat == CONSCIOUS) && P.client && P.mind && !P.mind.special_role)
+			Possible += P
+	if(!Possible.len)
+		user << "\red There doesn't appear to be anyone available for you to convert here."
+		return
+	var/mob/living/carbon/human/M = input("Select a person to convert", "Viva la revolution!", null) as mob in Possible
+	if(((user.mind in SSticker.mode.head_revolutionaries) || (user.mind in SSticker.mode.revolutionaries)))
```

Comment:
Just a suggestion: make this into a proc or a define, because its annoying to write out. For example:
```js
/proc/is_revolutionary(mob/M)
    return (M.mind in SSticker.mode.head_revolutionaries || M.mind in SSticker.mode.revolutionaries)
```

Reply:
Done, thanks for the tip.